gbpservice/nfp/orchestrator/drivers/orchestration_driver.py

```diff
@@ -995,24 +1021,24 @@ def plug_network_function_device_interfaces(self, device_data,
                             port['id'])
                         break
 
-            for port in device_data['ports']:
-                if port['port_classification'] == nfp_constants.CONSUMER:
-                    service_type = device_data[
-                        'service_details']['service_type'].lower()
-                    if service_type.lower() in [
```

Comment:
@ashutosh-mishra  while melding, this for loop got four spaces back and is leading to go out of the else block context so just moved in.